src/dependabot_alerts/cli.py

```diff
@@ -1,13 +1,52 @@
 from argparse import ArgumentParser
 from importlib.metadata import version
+from typing import List
```

Comment:
In modern versions of Python, this is deprecated in favor if `list[T]` - https://docs.python.org/3/library/typing.html#typing.List.

Reply:
Fixed